learningresources/tests/base.py

```diff
@@ -35,6 +35,13 @@ def logout(self):
         """just a client logout"""
         self.client.logout()
 
+    def assert_status_code(self, url, code, return_body=False):
+        """Asserts the status code of GET attempt"""
+        resp = self.client.get(url, follow=True)
+        self.assertTrue(resp.status_code == code)
+        if return_body:
```

Comment:
True. I'd rather get rid of the `if` and always return it, but not badly enough to figure out the perfect renaming now. Maybe in a future PR if I happen to be messing with the tests.